### PR TITLE
Move the tempest var to test_operator

### PR DESCRIPTION
**Pull request by raukadah**
Since we are already test_operator in all the edpm jobs.Let's use it here also.
    
    Note: it also includes a workaround for https://issues.redhat.com/browse/OSPCIX-487

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@
     vars:
       cifmw_operator_build_golang_ct: "docker.io/library/golang:1.20"
       cifmw_operator_build_golang_alt_ct: "quay.rdoproject.org/openstack-k8s-operators/golang:1.20"
-      cifmw_tempest_tempestconf_profile:
-          overrides:
+      cifmw_test_operator_tempest_config:
```
Comment (kopecmartin) [inline zuul.d/projects.yaml:9]: this doesn't seem right, you want to replace the variable meant for tempestconf, but cifmw_test_operator_tempest_config is the config for the whole operator, see it's default:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/test_operator/defaults/main.yml#L45

Comment (raukadah) [reply zuul.d/projects.yaml]: Done, the correct var is `cifmw_tempest_tempestconf_config`.

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -21,14 +21,14 @@
             # tempest gets configured with adminURL and that causes test
             # instability.
             identity.v3_endpoint_type: public
-      cifmw_tempest_tests_allowed:
+      cifmw_test_operator_tempest_include_list:
```
Comment (lpiwowar) [inline zuul.d/projects.yaml:24]: Both include and exclude list accept strings not list:)
```
cifmw_test_operator_tempest_include_list: |
  tempest.scenario
  tempest.scenario.test_minimum_basic.TestMinimumBasicScenario
```

File: zuul.d/projects.yaml [hunk 3]
```diff
@@ -6,8 +6,8 @@
     vars:
       cifmw_operator_build_golang_ct: "docker.io/library/golang:1.20"
       cifmw_operator_build_golang_alt_ct: "quay.rdoproject.org/openstack-k8s-operators/golang:1.20"
-      cifmw_tempest_tempestconf_profile:
-          overrides:
+      cifmw_tempest_tempestconf_config:
+          overrides: |
             compute-feature-enabled.vnc_console: true
```
Comment (lpiwowar) [inline zuul.d/projects.yaml:11]: The correct syntax is without the colon:
```
compute-feature-enabled.vnc_console true
```

File: zuul.d/projects.yaml [hunk 4]
```diff
@@ -21,14 +21,14 @@
...
-      cifmw_tempest_tests_allowed:
+      cifmw_test_operator_tempest_include_list:
       # NOTE(gibi): enable only the high level scenario tests to keep the
       # job run time reasonable
         - tempest.scenario
       # Plus an extra live migration test until we have cinder volumes / ceph
       # to run the live migration scenario tests with it
         - tempest.api.compute.admin.test_live_migration.LiveAutoBlockMigrationV225Test
-      cifmw_tempest_tests_skipped:
+      cifmw_test_operator_tempest_exclude_list:
```
Comment (lpiwowar) [inline zuul.d/projects.yaml:31]: ditto

File: zuul.d/jobs.yaml [hunk 5]
```diff
@@ -14,41 +14,43 @@
...
+        tempest.scenario.test_stamp_pattern.TestStampPattern
+        tempest.scenario.test_volume_boot_pattern.TestVolumeBootPattern
+        tempest.scenario.test_network_advanced_server_ops.TestNetworkAdvancedServerOps.test_server_connectivity_live_migration
+        tempest.scenario.test_server_volume_attachment.TestServerVolumeAttachScenarioOldVersion
+        tempest.scenario.test_server_volume_attachment.TestServerVolumeAttachmentScenario
+        tempest.scenario.test_instances_with_cinder_volumes.TestInstancesWithCinderVolumes
+        tempest.api.compute.admin.test_live_migration.LiveAutoBlockMigrationV225Test.test_live_block_migration_with_attached_volume
+      # Note(Chandan): Drop it once https://issues.redhat.com/browse/OSPCIX-487 fixes
+      cifmw_test_operator_tempest_extra_rpms:
+        - python3-testscenarios
```
Comment (karelyatin) [inline zuul.d/jobs.yaml:53]: for this to work we need to have repos setup, which i think we don't

can try with url instead https://trunk.rdoproject.org/centos9-antelope/deps/latest/noarch/python3-testscenarios-0.5.0-21.el9s.noarch.rpm

**Comment by karelyatin** [comment]
Let's get it in if the CI passes, workaround can be reverted once we have a new promotion with tempest fix included

File: zuul.d/jobs.yaml [hunk 6]
```diff
@@ -14,41 +14,43 @@
...
-        - tempest.scenario.test_server_volume_attachment.TestServerVolumeAttachScenarioOldVersion
-        - tempest.scenario.test_server_volume_attachment.TestServerVolumeAttachmentScenario
-        - tempest.scenario.test_instances_with_cinder_volumes.TestInstancesWithCinderVolumes
-        - tempest.api.compute.admin.test_live_migration.LiveAutoBlockMigrationV225Test.test_live_block_migration_with_attached_volume
+      cifmw_tempest_tempestconf_config:
+          # NOTE(alee) these tests will fail with barbican in the mix
+          # while cinder/nova is not configured to talk to barbican
+          # re-enable this when that support is added
+          # NOTE(gibi): This is a WA to force the publicURL as otherwise
+          # tempest gets configured with adminURL and that causes test
```
Comment (karelyatin) [inline zuul.d/jobs.yaml:22]: as moving to test operator would also need to add for mtu test to pass
      cifmw_test_operator_tempest_network_attachments:
        - ctlplane

File: zuul.d/jobs.yaml [hunk 7]
```diff
@@ -14,42 +14,42 @@
...
+        tempest.scenario.test_minimum_basic.TestMinimumBasicScenario
+        test_shelve_volume_backed_instance
+        tempest.scenario.test_stamp_pattern.TestStampPattern
+        tempest.scenario.test_volume_boot_pattern.TestVolumeBootPattern
+        tempest.scenario.test_network_advanced_server_ops.TestNetworkAdvancedServerOps.test_server_connectivity_live_migration
+        tempest.scenario.test_server_volume_attachment.TestServerVolumeAttachScenarioOldVersion
+        tempest.scenario.test_server_volume_attachment.TestServerVolumeAttachmentScenario
+        tempest.scenario.test_instances_with_cinder_volumes.TestInstancesWithCinderVolumes
+        tempest.api.compute.admin.test_live_migration.LiveAutoBlockMigrationV225Test.test_live_block_migration_with_attached_volume
+        # TODO (rlandy) remove when https://issues.redhat.com/browse/OSPCIX-126 is fixed
```
Comment (lpiwowar) [inline zuul.d/jobs.yaml:51]: I'm just wondering when whether we want to unskip this test since the linked ticked is fixed.

**Review by lpiwowar — APPROVED** [review]
Overall looks good to me :+1: 

https://logserver.rdoproject.org/38/738/48a99073afd8be31bb5995a403981ff8582e1d99/github-check/openstack-operator-tempest-multinode/69d8fce/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

**Pull request merged**